src/core/packages/chrome/layout/core-chrome-layout-components/layout_global_css.tsx

```diff
@@ -80,10 +82,17 @@ export const LayoutGlobalCSS = () => {
   `;
 
   const application = css`
+    ${layoutVarName('application.marginBottom')}: ${applicationMarginBottom}px;
+    ${layoutVarName('application.marginRight')}: ${applicationMarginRight}px;
+
     ${layoutVarName('application.top')}: ${bannerHeight + headerHeight}px;
-    ${layoutVarName('application.bottom')}: ${layoutVar('footer.height')};
```

Comment:
TODO for myself: we should refactor these variables to use static calculation as much as possible instead of dynamic `calc` for runtime perf reasons